packages/office-ui-fabric-react/src/components/Nav/Nav.tsx

```diff
@@ -118,7 +125,7 @@ export class Nav extends BaseComponent<INavProps, INavState> implements INav {
     );
   }
 
-  private _renderButtonLink(link: INavLink, linkIndex: number) {
+  private _renderButtonLink(link: INavLink, linkIndex: number, group: INavLinkGroup) {
```

Comment:
I need to be able to access the link's group's onLinkClick in _onNavAnchorLinkClicked and _onNavButtonLinkClicked. Unfortunately that requires passing it through a lot of other methods on the way.